base/doc/ltnews40.tex

```diff
@@ -384,46 +511,48 @@ \subsection{Fix wrong file type in a rollback warning}
 \end{verbatim}
 
 In some cases this message showed a wrong file type, i.e.,
-\verb|document class '<pkgname>'| or \verb|package '<clsname>'|.
-This has now been corrected.
+\verb|document class '<pkgname>'| or \verb|package '<clsname>'|.  This
+has now been corrected.
 %
 \githubissue{870}
 
 \subsection*{Fix existence check of document environments}
-\cs{NewDocumentEnvironment} and friends define (or redefine) a document
-environment using the space-trimmed \meta{envname}, but the existence
-check for \meta{envname} was done without space trimming. Thus when the
-user-specified \meta{envname} consists of leading and/or trailing
-space(s), it may lead to erroneously silent environment declaration.
-For example, in
+
+\cs{NewDocumentEnvironment} and friends define (or redefine) a
+document environment using the space-trimmed \meta{envname}, but the
+existence check for \meta{envname} was done without space
+trimming. Thus when the user-specified \meta{envname} consists of
+leading and/or trailing space(s), it may lead to erroneously silent
+environment declaration.  For example, in
 \begin{verbatim}
   \NewDocumentEnvironment{myenv}{}{begin}{end}
   \NewDocumentEnvironment{ myenv }{}{begin}{end}
 \end{verbatim}
 the first line defines a new environment \env{myenv} but the second
-line would check existence for \env{ myenv } (which is not yet defined),
-then redefine \env{myenv} environment without raising any errors.
-This has now been corrected.
+line would check existence for \env{ myenv } (which is not yet
+defined), then redefine \env{myenv} environment without raising any
+errors.  This has now been corrected.
 %
 \githubissue{1399}
 
 \subsection{Handling of global keys with spaces}
 
 If the global (class) options contained spaces around key names,
-\cs{ProcessKeyOptions} would fail to remove known keys from the
-list of unused global options and \cs{OptionNotUsed} would mistakenly
-add space-surrounded key names to that list.
-These has been corrected as a hotfix in patch level 1 of the November 2023
-release (but unfortunately not mentioned in~\cite{40:ltnews38}) and
-the current release, respectively.
+\cs{ProcessKeyOptions} would fail to remove known keys from the list
+of unused global options and \cs{OptionNotUsed} would mistakenly add
+space-surrounded key names to that list.  These has been corrected as
+a hotfix in patch level 1 of the November 2023 release (but
```

Comment:
```suggestion
space-surrounded key names to that list.  These have been corrected
as a hotfix in patch level 1 of the November 2023 release (but
```

Reply:
not quite, I think it should have been "This has" (the behavior changed = singular)

Reply:
There are two cases here.
- one for `\ProcessKeyOptions`, fixed in patch level 1 of the November 2023 release and
- another for `\OptionNotUsed`, fixed in the upcoming release.

Reply:
right you are, but then I think it should be written a bit differently to make it clearer.